src/sql/src/plan/statement/ddl.rs

```diff
@@ -2653,6 +2655,49 @@ pub fn plan_create_secret(
     }))
 }
 
+pub fn describe_create_connector(
+    _: &StatementContext,
+    _: &CreateConnectorStatement,
+) -> Result<StatementDesc, anyhow::Error> {
+    Ok(StatementDesc::new(None))
+}
+
+pub fn plan_create_connector(
+    scx: &StatementContext,
+    stmt: CreateConnectorStatement,
+) -> Result<Plan, anyhow::Error> {
+    scx.require_experimental_mode("CREATE CONNECTOR")?;
+
+    let create_sql = stmt.to_string();
```

Comment:
I think this wants to be like this from the secret code path:

```
let create_sql = normalize::create_statement(&scx, Statement::CreateSecret(stmt.clone()))?;
```

Reply:
Sounds good, I am now a little confused how I came up with this in all honesty.